README.md

```diff
@@ -56,36 +66,52 @@ module "eks_clickhouse" {
 ### Run Terraform to create the cluster
 
 Execute commands to initialize and apply the Terraform module. It will create an EKS cluster and install a ClickHouse sample database.
-```
+```sh
+export AWS_ACCESS_KEY_ID=<key-id>
+export AWS_SECRET_ACCESS_KEY=<super-secret-key>
+export AWS_SESSION_TOKEN="<session-token>"
+
 terraform init
 terraform apply
 ```
+
+By default it deploys to the `us-east-1` region, to set different region, use:
+```sh
+export AWS_REGION="eu-central-1"
```

Comment:
`AWS_REGION` is not needed

Reply:
it is. Otherwise, you get an error that subnets can't be provisioned.

Reply:
It's not... I deploy to different regions without setting any env var and is meant to work like that. If there is an error could be something else that needs to be fixed